components/eamxx/src/share/io/scream_io_utils.hpp

```diff
@@ -91,6 +100,79 @@ struct LongNames {
     {"hyam","hybrid A coefficient at layer midpoints"},
     {"hybm","hybrid B coefficient at layer midpoints"}
   };
+
+  // Create map of longnames, can be added to as developers see fit.
+  std::map<std::string,std::string> name_2_standardname = {
+    {"p_mid"                                                       , "air_pressure"},
+    {"p_mid_at_cldtop"                                             , "air_pressure_at_cloud_top"},
+    {"T_2m"                                                        , "air_temperature"},
+    {"T_mid"                                                       , "air_temperature"},
+    {"T_mid_at_cldtop"                                             , "air_temperature_at_cloud_top"},
+    {"aero_g_sw"                                                   , "asymmetry_factor_of_ambient_aerosol_particles"},
+    {"pbl_height"                                                  , "atmosphere_boundary_layer_thickness"},
+    {"precip_liq_surf_mass"                                        , "atmosphere_mass_content_of_liquid_precipitation"},
+    {"cldlow"                                                      , "cloud_area_fraction"},
```

Comment:
Are these really all the same?

Reply:
Yeah, I'm not sure.  I'm pretty sure I only grabbed standard_names I was able to confirm either on the CF website or from EAM.  Let me check again.

Reply:
Good catch, I found better standard names with a little more digging.